catalog-template/csm-operator-template.yaml

```diff
@@ -0,0 +1,26 @@
+      mediatype: image/png
+    name: dell-csm-operator
+    schema: olm.package
+  - entries:
+      - name: dell-csm-operator.v1.6.1
```

Comment:
Do we still need references for 1.6.1

Reply:
Yes, we would need that. This template file has the entries of all the CSM Operator versions that the user would see in the dropdown of versions of CSM Operator in OCP UI. Since we have removed upper bound for OCP from 1.6.1 operator, so operator version 1.6.1 would also be a part of the dropdown.